### ISSUE TITLE
Cannot apply a regex rule with negative lookbehind

### ISSUE THREAD

**Issue by ameshkov** [AdguardTeam/CoreLibs#19]
See `testNegativeLookBehindRegexpRule`

```
        final String ruleText = "/^https?\\:\\/\\/(?!(connect\\.facebook\\.net|ajax\\.cloudflare\\.com|www\\.google-analytics\\.com|ajax\\.googleapis\\.com|fbstatic-a\\.akamaihd\\.net|stats\\.g\\.doubleclick\\.net|api-secure\\.solvemedia\\.com|api\\.solvemedia\\.com|sb\\.scorecardresearch\\.com|www\\.google\\.com)\\/)/$script,third-party,xmlhttprequest";

        assertRuleMatches(true, ruleText, "http://traratatata.com/blahblah.js", "https://example.org/", null, null);
```

**Comment by ameshkov** [comment]
Fixed in 7cce29e3990b22946ae461e2337fbdd10d4fa10f

We still need you to add unit-tests on the C side.

**Comment by hmage** [comment]
C test added in f226778a9aefc206f67cd9aeabe6dae64661516e.
